src/Navbar.vue

```diff
@@ -1,10 +1,11 @@
 <template>
   <nav ref="navbar" :class="['navbar', 'navbar-expand-md',
   {
-    'navbar-dark':(type === 'inverse'),
-    'navbar-light':(type === 'default'),
-    'bg-dark':(type === 'inverse'),
-    'bg-light':(type === 'default')
+    'navbar-dark':(type === 'primary' || type === 'dark' || type !== 'none'),
```

Comment:
`type !== 'none'` includes `'light'` right?

Reply:
This has been fixed!